modules/ROOT/pages/restarting-a-node.adoc

```diff
@@ -3,76 +3,112 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-This topic describes the processes for removing an Anypoint Platform PCE node from active service, and returning it to service when ready.
+== Before You Begin
 
-== Notes
+* Set up and test the backup procedure on all clusters after you complete installation.
+* Perform a backup before attempting node maintenance to preserve data in the event of a failure.
 
-[NOTE]
-To preserve your data in case of system failure, ensure that your system is backed up before attempting node maintenance.
-[NOTE]
-The steps below must be performed on a single Anypoint Platform PCE node at a time.  If maintenance is required on more than one node, the entire process must be completed on a single node before the next node is addressed.
+== Cluster and Application Updates
 
-== Drain the Node
+The following steps must be performed on one Anypoint Platform Private Cloud Edition node at a time to minimize or eliminate component breakage during the update process. The entire process must be completed on each node before the next node is addressed. Verify the health status of all cluster components and applications after each node update before proceeding with the next node.
```

Comment:
Is there any way to emphasize this?

Reply:
If not, it's ok. but this is really important.

Reply:
Tagged it as important.

Reply:
is this okay now? not sure I got what's the request...